command/src/main/java/pw/chew/jdachewtils/command/OptionHelper.java

```diff
@@ -30,173 +30,198 @@
 import java.util.List;
 
 /**
- * A collection of useful methods for working with Options.
+ * Utility class containing useful methods for getting values of Slash command arguments.
+ * 
+ * <h2>Example</h2>
+ * <pre><code>
+ * public class MyCommand extends SlashCommand {
+ *     public MyCommand() {
+ *         this.name = "example";
+ *         this.help = "Example command";
+ *         
+ *         this.options = Arrays.asList(
+ *             new OptionData(OptionType.STRING, "string", "A String option").setRequired(true),
+ *             new OptionData(OptionType.USER, "user", "A optional User")
+ *         );
+ *     }
+ *     
+ *    {@literal @Override}
+ *     protected void execute(SlashCommandEvent event) {
+ *         String arg1 = OptionHelper.optString(event, "string", null);
+ *         User optionalUser = OptionHelper.optUser(event, "user", null);
```

Comment:
```suggestion
 *         // Get the provided user, or use the executor if they did not provide one
 *         User optionalUser = OptionHelper.optUser(event, "user", event.getUser());
```

Showcase the use of a default value maybe?